src/provider/qti.js

```diff
@@ -495,15 +495,21 @@ var qtiProvider = {
 
                 this.setState('closedOrSuspended', true);
 
+                let params = {
+                    itemDefinition: testContext.itemIdentifier,
+                    reason: {
+                        reasons: data && data.reasons,
+                        comment: data && (data.originalMessage || data.message)
+                    }
+                };
+
+                const itemState = self.itemRunner.getState();
+                if (Object.keys(itemState).length) {
```

Comment:
It's good that now we don't send `itemState` if we don't know it, but this condition still doesn't look better than `isCleared` - how does test-runner know it's '_null'_ state, not '_user removed all answers state_'? Also in reality it should work, so we can leave it, maybe at least move it to some helper function.
I don't have a suggestion on how to improve it if `context.state` and `context.itemSessionState` are still `interacting` and `unloadItem` method doesn't set any state.

Reply:
> Though, in reality it should work, so we can leave it. Maybe move it to some helper function.

Maybe here https://github.com/oat-sa/tao-test-runner-qti-fe/blob/master/src/helpers/currentItem.js  but I don't really know